test/utils/svm/validator.setup.ts

```diff
@@ -1,19 +1,58 @@
+import { SvmSpokeClient } from "@across-protocol/contracts";
+import { Address } from "@solana/kit";
 import { spawn } from "child_process";
-import path from "node:path";
 import fs from "node:fs/promises";
-import { SvmSpokeClient } from "@across-protocol/contracts";
+import path from "node:path";
+import tar from "tar";
 
+// Directory and file constants
 const LEDGER_DIR = path.resolve(__dirname, "..", ".ledger");
+const TARGET_DIR = path.resolve(__dirname, "..", "..", "..");
+const SVM_SPOKE_SO_PATH = path.resolve(TARGET_DIR, "target", "deploy", "svm_spoke.so");
+const BINARY_RELEASE_TAG = process.env.SVM_BINARY_RELEASE_TAG || "v4.0.11";
```

Comment:
might be better to try to resolve the installed contracts version and set the GitHub binary release tag accordingly

Reply:
@Reinis-FRP , I’ll follow your suggestion in a follow-up PR since that’ll need some logic updates now that we don’t have the enableRoutes function and deposit and fill both require a new delegate PDA.